src/docfx/lib/markdown/MarkdownEngine.cs

```diff
@@ -15,7 +15,9 @@
 using Markdig.Renderers.Html.Inlines;
 using Markdig.Syntax;
 using Markdig.Syntax.Inlines;
+using Microsoft.AspNetCore.Http.Features;
```

Comment:
Remove this using statement, markdown engine should not depend on anything AspNetCore.